spec.html

```diff
@@ -40423,7 +40418,7 @@ <h1>Map.prototype.set ( _key_, _value_ )</h1>
               1. Return _M_.
           1. If _key_ is *-0*<sub>𝔽</sub>, set _key_ to *+0*<sub>𝔽</sub>.
           1. Let _p_ be the Record { [[Key]]: _key_, [[Value]]: _value_ }.
-          1. Append _p_ to _entries_.
+          1. Append _p_ to _M_.[[MapData]].
```

Comment:
Might as well get rid of the `_entries_` aliases entirely at this point; it seems weirder to refer to the same value in two different ways in the same algorithm.

Reply:
Done.